profile/src/main/res/values/colors.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <color name="primaryColor">#FFBB86FC</color>
```

Comment:
these looks to be defined a few times. might be nice to have them just live in their own 'design' module or something